### DISCUSSION TITLE
When will the whisper-v3 model be released with Deepgram?

### DISCUSSION THREAD

**Discussion by mehdigreefhorst** [deepgram/community#420] [Feature Requests]
When will the whisper-v3 model be released with Deepgram?

**Comment by jkroll-deepgram** [comment]
Hi @mehdigreefhorst, we're glad to hear interest in whisper-v3. We are investigating Large-v3, and I'll be sure to keep this discussion thread updated if we add it to our model offerings. For the meantime, our own models are built in-house, trained by our own researchers, and are excellent (for pre-recorded audio as well as live-streaming).

**Comment by willemmulder** [comment]
Thanks for that @jkroll-deepgram! 

For context, I would always use Deepgram for english audio, but for Dutch that's a different story: in Whisper-v3 it's the highest scoring language while the Deepgram model has not had an update in a long time. 

I'm eager to see how Whisper-v3 performs and whether it can be made to support live-streaming! On huggingface, they seem to think that could work: https://huggingface.co/openai/whisper-large-v3

**Reply by jkroll-deepgram** [reply]
Hi @willemmulder, thanks for the feedback on how different models are working for you. Have you run any quick tests through Hugging Face yet to see what kind of change you notice personally with transcription using v3 versus v2? We're running our own benchmarking and not necessarily seeing our metrics lining up with the announcement; so make sure to try yourself rather than taking launch hype at face value.

That being said, I understand that we haven't updated our Dutch model recently, and naturally you'll be eager to use whichever model performs best for your use case. We're working on expanding our latest model architectures to more languages, so I hope we can bring you some exciting new releases in the coming months.

I see in HF's notes that they speculate that v3 could possibly be used for "near-real-time" applications; I'll defer that to other benchmarking and adaptations. Deepgram has purpose-built streaming models with exceptionally low latency.

**Reply by willemmulder** [reply]
Hi @jkroll-deepgram I ran some early tests, and in our usecases Whisper-v3 performs significantly better than previous models. So we're really eager to use the new models, maybe first Whisper and then Nova! Looking forward to it!

As a note, we are using live streaming, so that means that for Whisper we do chunking ourselves, using VAD detection etc. But we'd rather leave that to Deepgram and not worry about it :-)

**Reply by willemmulder** [reply]
@jkroll-deepgram Congrats on the Nova 2 release today! Good progress. Looking forward to Dutch in the coming months (y)

**Comment by willemmulder** [comment]
@jkroll-deepgram Any news on this yet? There is now also https://github.com/Vaibhavs10/insanely-fast-whisper that you might want to check out to make things more efficient!

**Reply by jkroll-deepgram** [reply]
Hi @willemmulder, thanks for checking in again. We are still continuing to look into Whisper v3, and have found some changes to the model implementation that make it non-trivial to support. Deepgram has its own word timing algorithms since Whisper does not natively provide timestamps, and v3 uses a new tokenizer, among other changes that will mean additional dev work to get it up and running to meet our bar of feature parity and accuracy.

Note that OpenAI is still itself [using v2 in its API](https://platform.openai.com/docs/guides/speech-to-text/quickstart). At this time, Deepgram isn't aiming to be a v3 early adopter; we have limited engineers and researchers, and we're more focused on training our own new models, including expanding Nova to more languages.

**Reply by willemmulder** [reply]
Hi @jkroll-deepgram that makes sense! I also read that there are some issues with v3 and 'hallucinations'. So we're also going to wait it out a little. And we keep looking forward to expanded Nova support for Dutch! :-)

**Reply by jkroll-deepgram** [reply]
Hi @willemmulder, I'm happy to finally share that we've released Nova-2 Dutch! You can use the parameters `model=nova-2&language=nl`. I'd appreciate any feedback and observations when you try it!

**Reply by willemmulder** [reply]
@jkroll-deepgram That's amazing! I'll try to test it out as soon as possible. In the meantime, would you be able to allow the use of nova-2 on the playground? https://playground.deepgram.com/?smart_format=true&language=nl&model=nova-2 currently redirects to the enhanced model instead of nova-2. Thanks so much! :-)

**Reply by jkroll-deepgram** [reply]
@willemmulder Thanks for the feedback, we've updated the Playground so Nova-2 Dutch is supported! Enjoy 🙂

**Reply by RazHadas** [reply]
Hi,
i want to use deepgram in Hebrew.
Anyone knows about support for whisper v3 in Hebrew ?

**Reply by jkroll-deepgram** [reply]
Hi @RazHadas, Deepgram still only serves Whisper v2 (same as OpenAI's own API). You can transcribe Hebrew with the URL parameters `language=he&model=whisper`.

**Reply by willemmulder** [reply]
It has been a while, but we're using Nova 2 for Dutch in production for a while now, and it works great! Just wanted to let you know 👍
